packages/server-boilerplate/src/utils/useForwardUnhandledRequests.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import { Express, NextFunction, Request, Response } from 'express';
+import { IncomingMessage, ServerResponse } from 'http';
+import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware';
```

Comment:
Do we need to move this dependency into `server-boilerplate` package.json?

Reply:
Yes - good catch thanks